src/renderer/components/hotbar/hotbar-icon.scss

```diff
@@ -56,6 +55,14 @@
     }
   }
 
+  &.disabled {
+    cursor: default;
+  }
+
+  &.inactive {
+    cursor: context-menu;
+  }
```

Comment:
This is also new, I found that because it was disabled I didn't realize (initially) that there was supposed to be a context menu.